sourcegraph/sourcegraph
# batches: Create a batch change from the search result page

Closes [#22352](https://github.com/sourcegraph/sourcegraph/issues/22352)

## Test plan


* Run a search query on the search page
* Ensure you have the flag `experimentalFeatures.batchChangesExecution` set to true
* The `Create Batch Change` should be available on the search page, once clicked it'll redirect you to create a batch change with the batch spec already pre-filled.
![CleanShot 2022-06-03 at 17 38 00@2x](https://user-images.githubusercontent.com/25608335/171897072-a0f0606a-d43a-47a5-936a-64e830973d50.png)
 

## App preview:

- [Web](https://sg-web-bo-implement-batch-change-from.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-tareuoxmiy.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
